site/en/about/bib.md

```diff
@@ -107,13 +107,13 @@ TensorFlow: Large-scale machine learning on heterogeneous systems,
 
 [Access this white paper.](https://www.usenix.org/system/files/conference/osdi16/osdi16-abadi.pdf)
 
-**Abstract:** TensorFlow is a machine learning system that operates at
+**Abstract:** TensorFlow is a machine learning system that operates at a 
```

Comment:
Can leave this as-is. It's idiomatic and would rather it match the published abstract: https://www.usenix.org/system/files/conference/osdi16/osdi16-abadi.pdf